src/Evolution/DiscontinuousGalerkin/Limiters/CMakeLists.txt

```diff
@@ -15,15 +19,36 @@ set(LIBRARY_SOURCES
   WenoType.cpp
   )
 
-add_spectre_library(${LIBRARY} ${LIBRARY_SOURCES})
+spectre_target_headers(
+  ${LIBRARY}
+  INCLUDE_DIRECTORY ${CMAKE_SOURCE_DIR}/src
+  HEADERS
+  HwenoImpl.hpp
+  Krivodonova.hpp
+  LimiterActions.hpp
+  Limiters.hpp  # just for namespace... include?
+  Minmod.hpp
+  Minmod.tpp
+  MinmodHelpers.hpp
+  MinmodTci.hpp
+  MinmodType.hpp
+  SimpleWenoImpl.hpp
+  Tags.hpp
+  Tci.hpp  # just for namespace... include?
```

Comment:
Should `spectre_target_headers` include Doxygen-driven headers like this (and the one above), that only serve to anchor the documentation of a namespace?

Reply:
I would include them because if someone eventually adds code to that file they will not likely remember to add it to this list...  @nilsdeppe thoughts?

Reply:
My original intent was to have literally all `cpp`, `tpp`, and `hpp` files being tracked by the build system